docs/index.html

```diff
@@ -282,8 +283,8 @@ <h3>TangleID </h3>
 								<span><i class="icon-globe"></i></span>
 							</div>
 							<div class="fh5co-post to-animate">
-								<h3>Scalable Data Business</h3>
-								<p>For the first time ever true micro and even nano-transactions are enabled due to the unique IOTA Tangle architecture, providing developers with a brand new set of tools for their applications in both IoT and Web.</p>
+								<h3>Scalability in Business</h3>
```

Comment:
"Scalability in Business" is not always meant to be scalable business, right?

Reply:
Not sure how to correct it, but I find it more wide-covered in that usage.